administrator/components/com_joomlaupdate/src/Model/NotificationModel.php

```diff
@@ -82,16 +82,45 @@ public function sendNotification($type, $oldVersion, $newVersion): void
             'url'        => Uri::root(),
         ];
 
+        // Determine the default admin language and load the language file for the fallback and default language
+        $defaultLocale   = ComponentHelper::getParams('com_languages')->get('administrator', 'en-GB');
+        $defaultLanguage = $app->getLanguage();
+        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, 'en-GB', true, true);
+        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $defaultLocale);
+
         // Send emails to all receivers
         foreach ($emailReceivers as $receiver) {
-            $params = new Registry($receiver->params);
-            $jLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, 'en-GB', true, true);
-            $jLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $params->get('admin_language', null), true, true);
+            $receiverParams = new Registry($receiver->params);
+            $receiverLocale = $receiverParams->get('admin_language', $defaultLocale);
+
+            // Temporarily set application language to user's language.
+            if ($receiverLocale !== $defaultLocale) {
+                $receiverLanguage = Factory::getContainer()
+                    ->get(LanguageFactoryInterface::class)
+                    ->createLanguage($receiverLocale, $app->get('debug_lang', false));
+
+                Factory::$language = $receiverLanguage;
 
-            $mailer = new MailTemplate('com_joomlaupdate.update.' . $type, $jLanguage->getTag());
+                if (method_exists($app, 'loadLanguage')) {
+                    $app->loadLanguage($receiverLanguage);
+                }
+
+                $receiverLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $receiverLocale);
+            }
+
+            $mailer = new MailTemplate('com_joomlaupdate.update.' . $type, $receiverLocale);
             $mailer->addRecipient($receiver->email);
```

Comment:
```suggestion
            $mailer->addRecipient($receiver->email, $receiver->name);
```
Pass name of the the recipient

Reply:
Not really the focus of my PR, but included it anyways